MV.Wildebeest.Core/source/core/java/co/mv/wb/framework/Util.java

```diff
@@ -32,4 +32,9 @@ public static String nameFromUri(String uri)
 
 		return uri.substring(index + 1);
 	}
+
+	public static String coalesceWhiteSpaces(String text){
+		return text.replaceAll("\\s+", " ").trim();
```

Comment:
Could you rename to coalesceWhitespace; also as a public method we should guard against null arguments - in this case if null is passed then the call to .replaceAll will throw a null reference exception.  There's examples you can follow of the style for null guards in other places in the code.

Reply:
Hey Aris - on further reflection - don't worry about the styling points.  I can't expect you to follow the style guide if its not written down so I'll get onto that asap.  However if you could - please add the null check and send me a new PR when you have a moment